packages/ketcher-react/src/script/ui/dialog/template/TemplateDialog.tsx

```diff
@@ -176,28 +176,15 @@ const TemplateDialog: FC<Props> = (props) => {
     )
   }
 
-  const result = (): Result | null => {
```

Comment:
Can't we remove `Result` interface?

Reply:
sure, thanks for the catch
[61ba157](https://github.com/epam/ketcher/pull/1892/commits/61ba157cc3b9b426ed8171bd35fcd3b7a543a8f3)